database/things/css.pldb

```diff
@@ -5,6 +5,7 @@ creators Håkon Wium Lie
 webRepl https://playcode.io/css/
 reference https://examples.p6c.dev/categories/parsers/CSSGrammar.html
 standsFor Cascading Style Sheets
+originCommunity Conseil Européen pour la Recherche Nucléaire
 
```

Comment:
country Unknown?

Reply:
It would be Swiss. CERN's mainly in Switzerland, and runs beneath the Swiss/French border.